desktop/package.json

```diff
@@ -58,42 +58,42 @@
   },
   "devDependencies": {
     "babel": "6.5.2",
-    "babel-cli": "6.10.1",
-    "babel-core": "6.10.4",
+    "babel-cli": "6.11.4",
+    "babel-core": "6.13.2",
     "babel-eslint": "6.1.2",
     "babel-loader": "6.2.4",
     "babel-plugin-react-transform": "2.0.2",
-    "babel-plugin-transform-runtime": "6.9.0",
-    "babel-preset-es2015": "6.9.0",
+    "babel-plugin-transform-runtime": "6.12.0",
+    "babel-preset-es2015": "6.13.2",
     "babel-preset-react": "6.11.1",
-    "babel-preset-stage-1": "6.5.0",
-    "babel-preset-stage-2": "6.11.0",
-    "babel-runtime": "6.9.2",
+    "babel-preset-stage-1": "6.13.0",
+    "babel-preset-stage-2": "6.13.0",
+    "babel-runtime": "6.11.6",
     "chai": "3.5.0",
     "chai-as-promised": "5.3.0",
     "del": "2.2.1",
-    "devtron": "1.2.1",
+    "devtron": "1.3.0",
     "electron-download": "git://github.com/keybase/electron-download#5350b5a1bad8ba7bb8c7201abe59ae3a03c3d75d",
-    "electron-packager": "7.3.0",
+    "electron-packager": "7.5.1",
```

Comment:
electron actually got the npm name 'electron', this supports when this changes later